src/nativestore/NodeManager.cpp

```diff
@@ -104,21 +108,32 @@ NodeManager::NodeManager(GraphConfig gConfig) {
                                  ", NodeBlock::BLOCK_SIZE: " + std::to_string(NodeBlock::BLOCK_SIZE));
         std::string errorMessage = "Node DB size does not comply to node block size Path = " + nodesDBPath;
         node_manager_logger.error(errorMessage);
-        throw std::runtime_error(errorMessage);
+    }
+    if (dbSize(relationsDBPath) % RelationBlock::BLOCK_SIZE != 0) {
+        node_manager_logger.warn("RelationsDB size: " + std::to_string(dbSize(relationsDBPath)) + ", RelationBlock::BLOCK_SIZE: " + std::to_string(RelationBlock::BLOCK_SIZE));
+        std::string errorMessage =
```

Comment:
Shall we make the two lines single one? I do not see any reason for assigning the error message to a string variable first.

```
node_manager_logger.error("RelationsDB size does not comply to node block size Path = " + relationsDBPath);
```

Reply:
Addresses with suggested approach

Reply:
Ok